source/clear-linux/tutorials/tutorial-proxy.rst

```diff
@@ -0,0 +1,103 @@
+.. _tutorial-proxy:
+
+Setting up proxy
+################
+
+This tutorial shows you how to configure your system for use behind a
+corporate proxy. 
+
+|CL| :ref:`tooling` applications already benefit from the :ref:`autoproxy`
+feature. 
+
+.. contents:: 
+    :local:
+    :depth: 1
+
+Prerequisites
+*************
+
+This tutorial assumes you have installed |CL| on your host system.
+For detailed instructions on installing |CL| on a bare metal system, visit
+the :ref:`bare metal installation guide <bare-metal-install-desktop>`.
+
+Shells and programs in a desktop session
+****************************************
+
+Terminal
+========
+
+Add the following to your ~/.bashrc:
+
+.. code-block:: bash
+
+    export http_proxy=http://your.http-proxy.url:port
+    export https_proxy=http://your.https-proxy.url:port
+    export no_proxy=".your-company-domain.com,localhost"
+
+wget
+****
+
+Run this command to enable downloading from websites from the terminal:
+
+.. code-block:: bash
+
+    echo >> ~/.wgetrc <<EOF
+    http_proxy = your.http-proxy.url:port
+    https_proxy = your.https-proxy.url:port
+    ftp_proxy = your.http-proxy.url:port
+    no_proxy = your-company-domain.com, localhost
+    EOF
+
+System service (Docker)
+***********************
+
+For Docker (and other services that use systemd), you can follow these steps to configure and check proxy settings:
```

Comment:
I suggest clarifying or adding a note here that this proxy is for the Docker service network connections (e.g. `docker pull`). Networking inside of a deployed container has different mechanisms for defining proxy information, depending on the image. (See: https://docs.docker.com/network/proxy/)